src/runner.rs

```diff
@@ -25,7 +25,48 @@ pub(crate) enum TickFlow {
     Exit,
 }
 
-pub(crate) const HELP_TEXT: &'static str = r"To start a transaction, enter 'start transaction' or 'begin'. To exit, enter 'exit' or press CTRL-D.";
+pub(crate) const HELP_TEXT: &'static str = r#"Shell Keys
+  Enter
+    - Runs the statement.
+  Escape+Enter (macOS)
+  Shift+Enter (Windows)
+    - Starts a new line to enter a statement that spans multiple lines. You can also copy input text with multiple lines and paste it into the shell.
+  Ctrl+C
+    - Cancels the current command.
+  Ctrl+D
+    - EOF / exit current level of shell. If not in a transaction, exit shell. If in a transaction, aborts the transaction.
+
+Database commands
+  start transaction
+  begin
+    - This starts a transaction.
+  commit
+    - This commits a transaction. If there is no transaction in progress, the shell reports an error saying that there is
```

Comment:
nit to make all the command descriptions have parallel construction:

>Commits a transaction. If there is no transaction in progress, the shell reports an error saying that there is no active transaction.

And remove "This..." from `abort` below as well:

>Aborts a transaction. If there is no transaction in progress, the shell reports an error saying that there is no active transaction.

Reply:
Updated